### PR TITLE
Teshari rework

### PR DESCRIPTION
**Pull request by Gr33d-y**
The first race I plan to update.
Current changes: They take 50% more damage from everything, have a higher chance for wounds, and a higher chance to loose limbs in combat. They have a better diet now and move 20% faster.

Let me be clear that this is not me making Teshari good and calling it a day, I plan to make all of our species feel unique and flavorful to play, teshari are just the start.

### CODE REVIEW

File: modular_skyrat/modules/teshari/code/_teshari.dm [hunk 1]
```diff
@@ -17,17 +17,21 @@
...
 		"legs" = "Normal Legs"
 	)
 	digitigrade_customization = DIGITIGRADE_NEVER
-	disliked_food = GROSS | GRAIN | GORE
-	liked_food = MEAT
+	disliked_food = GROSS | FRIED
+	liked_food = MEAT | GORE | SUGAR
 	changesource_flags = MIRROR_BADMIN | WABBAJACK | MIRROR_MAGIC | MIRROR_PRIDE | ERT_SPAWN | RACE_SWAP | SLIME_EXTRACT
 	payday_modifier = 0.75
+	speedmod = -0.2
```
Comment (Cyprex) [inline modular_skyrat/modules/teshari/code/_teshari.dm:33]: If this is a buff probably best not to do that.

Comment (Gr33d-y) [reply modular_skyrat/modules/teshari/code/_teshari.dm:32]: This is supposed to be a buff, as all species will get buffs to make them feel more flavorful. 
I've also upped the damage they take and their wounding rate and broken bones. Also, 20% sounds like a ton, but they really do not move all that much faster in the game.

File: modular_skyrat/modules/teshari/code/_teshari.dm [hunk 2]
```diff
@@ -17,17 +17,21 @@
 		TRAIT_ADVANCEDTOOLUSER,
 		TRAIT_CAN_STRIP,
 		TRAIT_LITERATE,
+		TRAIT_NOSOFTCRIT,
```
[the remark this answers is not in the document]
Comment (Gr33d-y) [reply modular_skyrat/modules/teshari/code/_teshari.dm]: This is being removed as it feels unneeded. They should be squishy.

File: modular_skyrat/modules/teshari/code/_teshari.dm [hunk 3]
```diff
@@ -17,17 +17,21 @@
...
 	)
 	digitigrade_customization = DIGITIGRADE_NEVER
-	disliked_food = GROSS | GRAIN | GORE
-	liked_food = MEAT
+	disliked_food = GROSS | FRIED
+	liked_food = MEAT | GORE | SUGAR
 	changesource_flags = MIRROR_BADMIN | WABBAJACK | MIRROR_MAGIC | MIRROR_PRIDE | ERT_SPAWN | RACE_SWAP | SLIME_EXTRACT
 	payday_modifier = 0.75
+	speedmod = -0.2
+	stunmod = 1.25
```
[the remark this answers is not in the document]
Comment (Gr33d-y) [reply modular_skyrat/modules/teshari/code/_teshari.dm:33]: Being stunned as one during testing made it feel like it did, however, if you can find a way to make them take more stun damage ill add that.

[the remark this answers is not in the document]
Comment (Gr33d-y) [reply modular_skyrat/modules/teshari/code/_teshari.dm:33]: Like, prolly 50% more stun damage

[the remark this answers is not in the document]
Comment (Gr33d-y) [reply modular_skyrat/modules/teshari/code/_teshari.dm]: > > Like, prolly 50% more stun damage
> 
> You can adjust the STAMINAMOD iirc, to make them take more stam damage.

this seems to not be a thing for some reason, at least when I tried adding it.

File: modular_skyrat/modules/teshari/code/_teshari.dm (hunk 1 above)
Comment (mcmeiler) [reply modular_skyrat/modules/teshari/code/_teshari.dm:32]: This is just stupid.

Teshari with this addition _cannot_ be outrun, and, therefore, chased, by anyone, barring security with disablers (_Which they rarely ever even get on maps to begin with_).

Comment (Gr33d-y) [reply modular_skyrat/modules/teshari/code/_teshari.dm:32]: > This is just stupid.
> 
> Teshari with this addition _cannot_ be outrun, and, therefore, chased, by anyone, barring security with disablers (_Which they rarely ever even get on maps to begin with_).

yes and no, Teshari, with this addition, can be chased. Yes, they are faster than most crew when unhurt, but if they are hurt or have their legs broken, they will be slowed down and easy to catch. Given these changes, it is not only easier to hurt them, it is easier to break ones leg or tear their muscles with just punches. Not to mention, if you are chasing one, due to the way that stations are laid out, often they will get to the door first and have to wait for it to open, this gives you enough time often to catch up quite easily. 

TLDR; Hurt teshari to slow them down if you need to, throw a chair at them, punch them, keep your mood up so you run faster.

Comment (mcmeiler) [reply modular_skyrat/modules/teshari/code/_teshari.dm:32]: > TLDR; Hurt teshari to slow them down if you need to, throw a chair at them, punch them, keep your mood up so you run faster.

So, what you are suggesting is for Sec to have a shotgun pass for Teshari who are running away, which they can't use half the time because the alert isn't high enough for them to carry lethal equipment. Raising alert for a single teshari on the station is also unreasonable.

File: modular_skyrat/modules/teshari/code/_teshari.dm (hunk 2 above)
[the remark this answers is not in the document]
Comment (mcmeiler) [reply modular_skyrat/modules/teshari/code/_teshari.dm]: Update the PR description to reflect this, then.

**Comment by Gr33d-y** [comment]
Closing this as I've realized that to do justice to most species I would need to be able to code far better than I can currently.
